rust/index/src/spann/utils.rs

```diff
@@ -525,6 +525,7 @@ pub async fn rng_query(
     normalized_query: &[f32],
```

Comment:
this is used on both read and write paths. We need to do this change only in the write path